src/components/policy/policy_regular/include/policy/policy_table/enums.h

```diff
@@ -89,6 +91,21 @@ enum Parameter {
   P_DEVICESTATUS,
   P_EMERGENCYEVENT,
   P_ECALLINFO,
+  P_ABS_STATE,
```

Comment:
These elements were not previously present, can you please further explain the changes?

Reply:
@mrapitis returned this changes back 6b4cb10d8136a5734041eca7d709d11e1dc7b317